packages/salesforcedx-vscode-core/src/commands/forceSourceStatus.ts

```diff
@@ -20,22 +21,28 @@ import {
 
 export enum SourceStatusFlags {
   Local = '--local',
-  Remote = '--remote'
+  Remote = '--remote',
 }
 
 export class ForceSourceStatusExecutor extends SfdxCommandletExecutor<{}> {
+  public command: string;
   private flag: SourceStatusFlags | undefined;
+  private description: string;
+  private logName: string;
 
-  public constructor(flag?: SourceStatusFlags) {
+  public constructor(flag?: SourceStatusFlags, cmdVersion?: CommandVersion) {
     super();
     this.flag = flag;
+    this.command = `force:source:${cmdVersion? cmdVersion : CommandVersion.Default}:status`;
+    this.description = ['force_source_status', cmdVersion, 'text'].filter(Boolean).join('_');
+    this.logName = ['force_source_status', cmdVersion].filter(Boolean).join('_');
```

Comment:
whoa...what's this do?

Reply:
It adds the `_` separator only if the vars in the array are all present - yeah its a bit wonky logic :) Its removed now - moved them to static values instead of computing them dynamically and incurring the complexity/overhead